src/Nox.Docs/Templates/EntityDomainEvents.template.md

```diff
@@ -3,7 +3,7 @@
 This document provides information about the {{entity.Name}} Domain Events in our application.
 
 ## Events
-{{ if entity.Persistence.Create.RaiseEvents }}
+{{ if entity.Persistence.Create.RaiseEvents == "DomainEventsOnly" ||  entity.Persistence.Create.RaiseEvents == "DomainAndIntegrationEvents" }}
```

Comment:
see the entity.template comment, can be applied here also

Reply:
Changes have been pushed.